docs/client.rst

```diff
@@ -72,18 +72,111 @@ WBEM Operation recording
+Setting the value of TestClientRecorder into this
+attribute activated and enabled the recorder.  Setting it to None
+deactivated the recorder.
+
+Effective version 0.11.0, pywbem added a second recorder and the requirement
+that both recorders operate simultaneously. If both are active and
+enabled, operation data can be both recorded to a yaml file and to the log
+simultaneously.
+
+The method to activate has therefore changed. To activate a recorder you must
+use the WBEM connection method:
+
+:method:`~pywbem.WBEMConnection.add_operationsrecorder()`.
+
+This adds the defined recorder to a list of active recorders in WBEMConnection.
+All active recorders are called for each WBEMConnection method that sends
+information to the WBEM server.
+
+The recorder can be  be disabled with the disable() method of
+:method:`~pywbem.BaseOperationRecorder.disable` and enabled with the
+:method:`~pywbem.enable` method.
+
+In addition, the logger names for the operations and http loggers should be
+defined to create usable loggers since the pywbem loggers use attributes in
+addition to the standard python loggers.  This is done with the
+:class:`~pywbem.PywbemLoggers` and specifically with the method
+:method:`~pywbem.PywbemLoggers.create_logger` which creates one or more loggers
+from the input parameters.
+
+::
+
+    logger.setLevel(logging.DEBUG)
+    # To add the LogOperationRecorder
+    # Enable the logs using one of the methods in PywbemLoggers
+    PywbemLoggers.create_logger('all')   # enables both loggers
+    # Create the connection and enable the logger
+    conn = WBEMConnection(...)
+    logging = LogOperationRecorder()
+    add_operations_recorder(LogOperationRecorder())
+    # The LogOperationRecorder will not be active.
+    # To enable writing logs
+
+
+    # To add both recorders
+    logger.setLevel(logging.DEBUG)
+    PywbemLoggers.create_logger('all')   # enables both loggers
+    conn = WBEMConnection(...)
+    add_operations_recorder(LogOperationRecorder())
+    yamlfp = TestClientRecorder.open_file(self.yamlfile, 'a')
+    add_operations_recorder(TestClientRecorder(yamlfp))
+
+    # Both TestClientRecorder and LogOperationRecorder are be
+    # active, enabled and recording
+    # To change the enabled state of either, use the enabled/disabled
+    # methods of the Recorder
+
+    logging.disable()   # disables recording to the log
+
+Activated loggers can computing-wise expensive so it is best not to activate
+either logger unless they are to be used for that specific WBEMConnection.
+
+The Recorder enable() and disable() methods simply set flags to bypass creating
+the final recorder output so activating and disabling is still more
+compute-wise expensive than not activating a recorder at all.
+
+The :class:`~pywbem.LogOperationRecorder' is a more complete mechanism than the
+:attr:`~pywbem.WBEMConnection.debug` in that it records
+information on operations and http interactions with a WBEM server to a python
```

Comment:
http -> HTTP

Reply:
the `http` should be lower class. Found and fixed one place in text that lower case used.